package.json

```diff
@@ -8,10 +8,11 @@
     "npm": "^3.0.0"
   },
   "scripts": {
+    "build:autoprefix": "postcss --use autoprefixer --autoprefixer.browsers 'last 3 versions' -o dist/mofo-ui.css dist/mofo-ui.css",
```

Comment:
due to how not-cross-platform autoprefixer is, this only works *without* the `--autoprefixer.browsers 'last 3 versions'` part: the browser version flag relies on bash-specific syntax that breaks all the things in both windows command shell as well as Powershell =(